app/enterprise/2.1.x/developer-portal/administration/application-registration/azure-oidc-config.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Set up External Portal Application Authentication with Azure AD and OIDC
+---
+
+## Overview
+
+These instructions help you set up Azure AD as your third-party identity provider
+for use with the Kong OIDC and Portal Application Registration plugins.
+
+### Prerequisites
+
+- The `portal_app_auth` configuration option is configured for your OAuth provider
+  and strategy (`kong-oauth2` or `external-oauth2`). See
+  [Configure the Authorization Provider Strategy](/enterprise/{{page.kong_version}}/developer-portal/administration/application-registration/#portal-app-auth) for the Portal Application Registration plugin.
+
+## Create an Application in Azure
+
+1. Within Azure, go to the **App Registrations** service and register a new application.
+
+   ![Azure App Registrations](/assets/images/docs/dev-portal/ms-azure-app-reg.png)
+
+1. In **Certificates & secrets**, create a Client secret and save it in a
+   secure location. You can only view the secret once.
+
+1. Under **Manifest**, update `accessTokenAcceptedVersion=2` (default is null).
+   The JSON for your application should look similar to this example:
+
+  ![Azure Manifest](/assets/images/docs/dev-portal/azure-manifest.png)
+
+## Create a Service and a Route in Kong
+
+1. Create a Service example:
+
+   ```bash
+   http put :8001/services/httpbin-service-azure url=https://httpbin.org/anything
+   ```
+1. Create a Route example:
+
+   ```bash
+   http -f put :8001/services/httpbin-service-azure/routes/httpbin-route-azure paths=/httpbin-azure
+   ```
+
+## Map the OIDC and Application Registration Plugins to the Service
+
+Map the OpenID Connect and Application Registration plugins to the **Service**.
+The plugins must be applied to a Service to work properly.
+
+1. Configure the OIDC plugin for the Service:
+
+   ```bash
+   http -f :8001/services/httpbin-service-azure/plugins name=openid-connect config.issuer=https://login.microsoftonline.com/<your_tenant_id>/v2.0 config.display_errors=true config.client_id=<your_client_id> config.client_secret="<your_client_secret>" config.consumer_claim=aud config.scopes=openid config.scopes=<your_client_id>/.default
+   ```
+
+1. Configure the Application Registration plugin for the Service:
+
+   ```bash
+   http -f :8001/services/httpbin-service-azure/plugins name=application-registration config.auto_approve=true config.display_name="For Azure" config.description="Uses consumer claim with various values (sub, aud, etc.) as registration id to support different flows and use cases." config.show_issuer=true
+   ```
+
+1. Get an access token using the Client Credential workflow and convert the token
+   into a JSON Web Token (JWT). You can get the token from Azure or Kong. Replace
+   the placeholder values for `<your_tenant_id>`, `<your_client_id>`, and `<your_client_secret>`.
+
+   - Get a token from Azure:
```

Comment:
We currently have a bug in the output that reads unordered lists as implicit entries in an ordered list, so the numbering is broken. Change this entry and the following one to either a definition list or a numbered list (see https://deploy-preview-2300--kongdocs.netlify.app/hub/plugins/overview/#terminology for example formatting for a definition list).

Reply:
> We currently have a bug in the output that reads unordered lists as implicit entries in an ordered list, so the numbering is broken. Change this entry and the following one to either a definition list or a numbered list (see https://deploy-preview-2300--kongdocs.netlify.app/hub/plugins/overview/#terminology for example formatting for a definition list).

@lena-larionova I didn't even notice this at first. I just fixed this issue in another ticket for the dev portal. So this is an either or situation, so ul was perfect, numbered list is not appropriate here. Will this bug be fixed in the future? I could just use indents in the interim, or defn list.

Reply:
Yes, it'll be fixed in the future, once we either get Vlad to do it or I figure it out. Indents or definition list will work in the interim.